### PR TITLE
S02 us1 homepage first section

### PR DESCRIPTION
**Pull request by PaulBOURY**
first version of the first section in the homepage. We need to present something

### CODE REVIEW

**Review by pjroggy — APPROVED** [review]

File: client/src/components/ClickableLogo.jsx
Comment (FGAT3) [inline client/src/components/ClickableLogo.jsx:1]: Don't need a component

File: client/src/components/ClickableLogo.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  display: block;
+  margin: auto;
+  width: calc(100% - 1rem);
+
+  @include for-desktop-up {
+    width: calc(35% - 1rem);
+  }
+}
+
+h1 {
```
Comment (FGAT3) [inline client/src/components/ClickableLogo.scss]: Will be inside index.scss

File: client/src/assets/images/Logo-WG.svg
Comment (FGAT3) [inline client/src/assets/images/Logo-WG.svg:1]: Plz delete this image and import the existing one in "server/public/assets/images/logo/logo-wildy-gamy-wg.svg"

File: client/src/assets/images/logo-wild-gamy-v3.svg
Comment (FGAT3) [inline client/src/assets/images/logo-wild-gamy-v3.svg:1]: do you also need this image ?

File: client/src/components/ClickableLogo.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+import LogoSrc from "../assets/images/logo-wild-gamy-v3.svg"
```
Comment (FGAT3) [inline client/src/components/ClickableLogo.jsx]: To import from the backend use this :

const BASE_URL = import.meta.env.VITE_API_URL;
const logo-wildy-gamy-wg = `${BASE_URL}/logo/logo-wildy-gamy-wg.svg`;

Or logoWildyGamyWg if eslint don't like kebab case

**Review by FGAT3 — CHANGES_REQUESTED** [review]

File: client/src/components/ClickableLogo.jsx
Comment (PaulBOURY) [reply client/src/components/ClickableLogo.jsx:1]: everything is in HomePage now

File: client/src/components/ClickableLogo.scss (hunk 1 above)
Comment (PaulBOURY) [reply client/src/components/ClickableLogo.scss]: done

File: client/src/components/ClickableLogo.jsx (hunk 2 above)
Comment (PaulBOURY) [reply client/src/components/ClickableLogo.jsx]: i quite don't understand this so we'll see that on monday if that's okay

**Review by FGAT3 — APPROVED** [review]
GG

**Review by JadeJolivet — APPROVED** [review]

**Review by Elea641 — APPROVED** [review]

**Pull request merged**